docs/src/user-guide/getting-started.md

```diff
@@ -24,6 +24,10 @@ You can install and configure ESLint using this command:
 
 ```shell
 npm init @eslint/config
+
+# or
+
+yarn dlx @eslint/create-config
```

Comment:
This was intentional we didn't want to add all package managers for initializing the config. https://github.com/eslint/eslint/pull/16084#discussion_r915359596

Reply:
Then we should probably remove the yarn command here too:
https://github.com/eslint/eslint/blob/3cbe11f260c0206cb422778fef4aa85433b389eb/docs/src/user-guide/getting-started.md?plain=1#L35-L43

Reply:
Yea good catch, may be we can remove it. But would like to hear from others as well :)

Reply:
The installation case is different than the execution case. For installation, the command will ask if you want to install using npm, yarn, pnpm, etc. so it doesn’t really matter how you run it. I think for executing eslint it make sense to also mention yarn.